### PR TITLE
[#267,irods/irods#7495] Update for MySQL 8.4

### PR DESCRIPTION
**Pull request by SwooshyCueb**
Addresses #267
Addresses irods/irods#7495

### CODE REVIEW

**Review by alanking — APPROVED** [review]
Seems good to me. Let's get an approval from @korydraughn and/or @trel before #s.

File: docs/getting_started/installation.md [hunk 1]
```diff
@@ -79,10 +79,13 @@ N rows in set (0.00 sec)
 ~~~
 
 !!! Note
-    MySQL v5.7 uses a default transaction isolation level of `REPEATABLE-READ`. You'll need to change this to `READ-COMMITTED` so that iRODS operates properly. This setting needs to be set globally. One way to achieve this is by starting MySQL with `--transaction-isolation=READ-COMMITTED`. For more information, see [https://dev.mysql.com/doc/refman/5.7/en/innodb-transaction-isolation-levels.html](https://dev.mysql.com/doc/refman/5.7/en/innodb-transaction-isolation-levels.html).
-    
+    iRODS is incompatible with versions 8.1, 8.2, and 8.3 of the MySQL ODBC Connector. You'll need to use v8.0 or v8.4.
```
Comment (trel) [inline docs/getting_started/installation.md:82]: versions -> "unreleased versions" ?

Comment (korydraughn) [reply docs/getting_started/installation.md:82]: Where does it say they're unreleased?
Is it because they aren't considered GA (General Availability)?

**Review by korydraughn — APPROVED** [review]
Looks good to me.

Let's wait for @trel's response/approval.

File: docs/getting_started/installation.md (hunk 1 above)
Comment (SwooshyCueb) [reply docs/getting_started/installation.md:82]: Yeah they're released. They're just not considered stable or GA.

Comment (trel) [reply docs/getting_started/installation.md:82]: Ah, then carry on.  Resolving.

**Comment by trel** [comment]
I'm go.

**Review by trel — APPROVED** [review]

**Comment by SwooshyCueb** [comment]
\#'d

**Pull request merged**